### ISSUE TITLE
Browser should not lose focus when a custom ad notification is displayed

### ISSUE THREAD

**Issue by tmancey** [brave/brave-browser#15613]
## Description 

Browser should not lose focus when a custom ad notification is displayed

## Steps to Reproduce

   1. Launch browser
   2. Click in the address bar
   3. Wait for ad notification to be displayed
   4. Type on the keyboard

## Actual result:

The user is unable to type in the address bar as the browser window has lost focus

## Expected result:
User should be able to type in the address bar

## Reproduces how often: 

Easily reproduced

## Version/Channel Information:

- Can you reproduce this issue with the current release? N/A
- Can you reproduce this issue with the beta channel? N/A
- Can you reproduce this issue with the nightly channel? Yes

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? N/A
- Does the issue resolve itself when disabling Brave Rewards? N/A
- Is the issue reproducible on the latest version of Chrome? N/A

**Comment by bsclifton** [comment]
Ran into this yesterday; just seeing the issue now 😄 Nice fix, @tmancey!

cc: @rebron @stephendonner - this is the issue I had mentioned in triage 😄

**Comment by stephendonner** [comment]
Verified `FIXED` using 


Brave | 1.26.11 Chromium: 90.0.4430.93 (Official Build) nightly (x86_64)
-- | --
Revision | 4df112c29cfe9a2c69b14195c0275faed4e997a7-refs/branch-heads/4430@{#1348}
OS | macOS Version 11.3.1 (Build 20E241)

Steps:

1.  launched browser
2.  enabled Rewards; skipped onboarding tour
3.  opened a few new-tab pages
4.  typed `abc` in the URL bar
5.  waited for ad notification to be displayed
4.   resume typing on the keyboard

Confirmed I was able to keep typing while the ad-notification popup was still displayed, without needing to re-focus the URL bar using `command` + `L`

![focus-issue](https://user-images.githubusercontent.com/387249/117508423-665a6c80-af3d-11eb-8a6d-ba68d69fbddd.gif)
